setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name="django-simple-bulma",
-    version="1.1.4",
+    version="1.1.3",
```

Comment:
bump this to 1.1.5 instead, we can't have releases go back in time like that :smile: